processing/format_qaqc_driver.py

```diff
@@ -390,25 +416,24 @@ def run(self):
                                   f'   - Message: {msg}')
                         _log.debug('[EMAIL AMP] Sending email '
                                    f'to AMP for token: {token}')
-                        self.send_email_to_amp(msg)
+                        self.send_email_to_amp(msg, token, site_id)
```

Comment:
You seem to be making the same `send_email_to_amp` call in both situations, why not put it at the end?

Reply:
I refactor the logic a bit and group this function call to outside